### PR TITLE
Update Avg Conditioned Floor Area

### PR DESCRIPTION
**Pull request by rajeee**
## Pull Request Description

Address: #1273 
Average Conditioned Floor Area before the change: 1651 sq ft
Average Conditioned Floor Area after the change: 1490 sq ft

**Related PRs:** 
- https://github.com/NREL/resstock-estimation/pull/424
- https://github.com/NREL/resstock-estimation/pull/432

## Checklist

Not all may apply:

~~- [ ] Tests (and test files) have been updated~~
~~- [ ] Documentation has been updated~~
~~- [ ] If related to resstock-estimation, checklist includes [data dictionary](https://github.com/NREL/resstock/tree/develop/resources/data/dictionary), [source report](https://github.com/NREL/resstock/tree/develop/project_national/resources/source_report.csv), [options saturation](https://github.com/NREL/resstock/tree/develop/project_national/resources/options_saturations.csv), [options_lookup](https://github.com/NREL/resstock/blob/develop/resources/options_lookup.tsv).~~
- [x] Add to the [changelog_dev.rst file](https://github.com/NREL/resstock/tree/develop/docs/read_the_docs/source/changelog/changelog_dev.rst)
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected regression test changes on CI (checked comparison artifacts)

### CODE REVIEW

**Comment by rajeee** [comment]
Slight decrease in energy use.
image

File: measures/ResStockArguments/measure.rb [hunk 1]
```diff
@@ -404,43 +404,43 @@ def run(model, runner, user_arguments)
...
+               ['2500-2999', HPXML::ResidentialTypeSFD] => 2451,  # From RECS2020
+               ['2500-2999', HPXML::ResidentialTypeSFA] => 2299,  # From RECS2020
+               ['2500-2999', HPXML::ResidentialTypeApartment] => 2818,  # From RECS2020
+               ['2500-2999', HPXML::ResidentialTypeManufactured] => 2616,  # From RECS2020
+               ['3000-3999', HPXML::ResidentialTypeSFD] => 2998,  # From RECS2020
+               ['3000-3999', HPXML::ResidentialTypeSFA] => 3077,  # From RECS2020
+               ['3000-3999', HPXML::ResidentialTypeApartment] => 3106,  # From RECS2020
+               ['3000-3999', HPXML::ResidentialTypeManufactured] => 3000,  # From RECS2020
+               ['4000+', HPXML::ResidentialTypeSFD] => 4277,  # From RECS2020
+               ['4000+', HPXML::ResidentialTypeSFA] => 3795,  # From RECS2020
```
Comment (afontani) [inline measures/ResStockArguments/measure.rb:440]: @lixiliu saw that this floor area is not larger than 4000 ft2.

File: measures/ResStockArguments/measure.rb [hunk 2]
```diff
@@ -404,43 +404,43 @@ def run(model, runner, user_arguments)
...
+               ['1500-1999', HPXML::ResidentialTypeManufactured] => 1615,  # From RECS2020
+               ['2000-2499', HPXML::ResidentialTypeSFD] => 2062,  # From RECS2020
+               ['2000-2499', HPXML::ResidentialTypeSFA] => 1983,  # From RECS2020
+               ['2000-2499', HPXML::ResidentialTypeApartment] => 2135,  # From RECS2020
+               ['2000-2499', HPXML::ResidentialTypeManufactured] => 2082,  # From RECS2020
+               ['2500-2999', HPXML::ResidentialTypeSFD] => 2451,  # From RECS2020
+               ['2500-2999', HPXML::ResidentialTypeSFA] => 2299,  # From RECS2020
+               ['2500-2999', HPXML::ResidentialTypeApartment] => 2818,  # From RECS2020
+               ['2500-2999', HPXML::ResidentialTypeManufactured] => 2616,  # From RECS2020
+               ['3000-3999', HPXML::ResidentialTypeSFD] => 2998,  # From RECS2020
```
Comment (afontani) [inline measures/ResStockArguments/measure.rb:435]: Less than 3,000 ft2.

File: measures/ResStockArguments/measure.rb [hunk 3]
```diff
@@ -404,43 +404,43 @@ def run(model, runner, user_arguments)
...
+               ['1000-1499', HPXML::ResidentialTypeSFD] => 1171,  # From RECS2020
+               ['1000-1499', HPXML::ResidentialTypeSFA] => 1161,  # From RECS2020
+               ['1000-1499', HPXML::ResidentialTypeApartment] => 1119,  # From RECS2020
+               ['1000-1499', HPXML::ResidentialTypeManufactured] => 1177,  # From RECS2020
+               ['1500-1999', HPXML::ResidentialTypeSFD] => 1596,  # From RECS2020
+               ['1500-1999', HPXML::ResidentialTypeSFA] => 1571,  # From RECS2020
+               ['1500-1999', HPXML::ResidentialTypeApartment] => 1603,  # From RECS2020
+               ['1500-1999', HPXML::ResidentialTypeManufactured] => 1615,  # From RECS2020
+               ['2000-2499', HPXML::ResidentialTypeSFD] => 2062,  # From RECS2020
+               ['2000-2499', HPXML::ResidentialTypeSFA] => 1983,  # From RECS2020
```
Comment (afontani) [inline measures/ResStockArguments/measure.rb:428]: Less than 2,000 ft2

**Comment by afontani** [comment]
Options:
1. Use CFA across the board
2. Re-specify the differences in assessed and conditioned floor area in the documentation.

Maybe #2. Lixi brought up that the NEC calcs use the conditioned floor area.

**Comment by afontani** [comment]
Lixi: Commented that NEC calculations uses FFA, so might need both for panels work in FY25. 
Janet: Since there are already both, then maybe maintain both. Update names for clarity? Update data dictionaries if we understand what is happening. 
Scott: There are different floor areas in OS-HPXML schema. We could assign different elements for clarity.
- GrossFloorArea
- NetFloorArea
- ConditionedFloorArea
- FinishedFloorArea
- CooledFloorArea
- HeatedFloorArea
- UnconditionedFloorArea

Action: Tony to make a little write up about where things are now and circulate it.

**Comment by afontani** [comment]
After making the change, the floor area generally decreased. There area a few instances where the floor area increased a little bit. The change impacted a few end-uses: Heating, Cooling, Lighting, and Plug loads.

Screenshot 2024-10-01 at 9 28 22 AM
Screenshot 2024-10-01 at 9 27 55 AM
Screenshot 2024-10-01 at 9 27 38 AM
Screenshot 2024-10-01 at 9 24 25 AM

**Comment by rajeee** [comment]
Changes in conditioned floor area distribution for a 550K national sample.

National Average Conditioned Floor Area Before (v1): 1650.78 sqft
National Average Conditioned Floor Area After (v2): 1489.81 sqft

This PR introduces dependency on whether heated basement exists or not.
image

**Comparison to existing mapping**
image

The conditioned floor area is reduced more substantially for SFA and SFD because the floor area bin can include basement area even if they are not heated. 

**Distribution comparison**

image

If we zoom-in in y-axis, the SFA and SFD can be seen to be split in two bands - one band each based on whether the building has heated basement or not.

image

**Comment by afontani** [comment]
A couple of data-points:
- National Average Conditioned Floor Area Before (v1): 1650.78 sqft
- National Average Conditioned Floor Area After (v2): 1489.81 sqft
- AHS 2023 Mean: 1,784 sqft
- RECS 2023 minus heating/cooled garage: 1651 sqft

A summary table of what spaces are included in various sources:
Screenshot 2024-10-01 at 1 18 11 PM
C = Conditioned, U=Unconditioned

**Pull request closed without merging**